OSPSuite.Presentation.Importer/Core/IDataSet.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System.Collections.Generic;
+
+namespace OSPSuite.Presentation.Importer.Core
+{
+   /// <summary>
+   /// Data from a single experiment
+   /// </summary>
+   public interface IDataSet
```

Comment:
Maybe we do not need an interface for every single domain object we are creating?